test/deploy/kubernetes/manual/deploy.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+SCRIPT_DIR=$(dirname $0)
+MAXNODES="3"
+NAMESPACE="cortx"
+
+function print_header {
+    echo -e "---------------------------------------------------"
+    echo -e "$1"
+    echo -e "---------------------------------------------------"
+}
+
+# Create NameSpace
+kubectl create namespace $NAMESPACE
+
+# Create ConfigMap
+kubectl create configmap solution-config \
+    --from-file=$SCRIPT_DIR/solution-config/cluster.yaml \
+    --from-file=$SCRIPT_DIR/solution-config/config.yaml \
+    --namespace $NAMESPACE
+
+# Create Persistent Volumes for Block Devices
+for NODE_INDEX in $(seq 1 $MAXNODES); do
+    NODE_NAME="node"$NODE_INDEX;
+    print_header "Creating Persistent Volumes - $NODE_NAME"
+    NODE_PVOL=$SCRIPT_DIR/persistent-volumes/block_devices_$NODE_NAME.yaml;
```

Comment:
Change this:
`NODE_BDEV=$SCRIPT_DIR/persistent-volumes/block_devices_$NODE_NAME.yaml;`